model/GlobalItems.inc.php

```diff
@@ -81,21 +81,48 @@ public static function getGlobalItems($params) {
 		StatsD::timing("api.globalitems", $time * 1000);
 		
 		$code = curl_getinfo($ch, CURLINFO_HTTP_CODE);
-		
 		if ($code != 200) {
-			throw new Exception($code . " from Global Items server "
-				. "[requestURL: '$requestURL'] [RESPONSE: '$response']");
+			throw new Exception($code . " from global items server "
+				. "[URL: '$requestURL'] [RESPONSE: '$response']");
+		}
+		
+		$info = curl_getinfo($ch);
+		$headerSize = $info['header_size'];
+		$body = trim(mb_substr($response, $headerSize));
+		$rawHeaders = explode("\n", trim(mb_substr($response, 0, $headerSize)));
+		unset($rawHeaders[0]);
+		$headers = [];
+		foreach ($rawHeaders as $line) {
+			list($key, $val) = explode(':', $line, 2);
+			$headers[strtolower($key)] = trim($val);
 		}
```

Comment:
I guess this would work, but technically looking for `\r\n\r\n` and `\r\n` would be the stricter way to do this, no?

E.g., https://stackoverflow.com/a/10590242